jenkins/settings.xml

```diff
@@ -81,14 +87,31 @@
     </profile>
     <profile>
       <id>deploy-to-urm</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
       <properties>
           <altDeploymentRepository>snapshots::default::${env.URM_URL}-local</altDeploymentRepository>
       </properties>
      </profile>
+     <!-- Add Sonatype staging repo for buiding rapids against pre-release jars  -->
+     <profile>
+      <id>sonatype-staging</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <repositories>
+        <repository>
+           <id>sonatype.staging</id>
+           <url>https://oss.sonatype.org/content/repositories/staging</url>
+           <releases>
+             <enabled>true</enabled>
+           </releases>
+           <snapshots>
+             <enabled>false</enabled>
+           </snapshots>
+        </repository>
+      </repositories>
+     </profile>
   </profiles>
-  <activeProfiles>
```

Comment:
Remove `activeProfiles`, seems maven -P!xxx not works for `activeProfiles` items
change to use:
     <activation>
        <activeByDefault>true</activeByDefault>
      </activation>